src/browserlib/extract-cssdfn.mjs

```diff
@@ -607,7 +607,17 @@ const extractTypedDfn = dfn => {
       // Don't attempt to parse pre tags at this stage, they are tricky to
       // split, we'll parse them as text and map them to the right definitions
       // afterwards.
-      res = { name: getDfnName(dfn) };
+      // That said, we may be looking at a function definition on the right hand
+      // side of a production rule, as in the definition of "linear()" in
+      // css-easing-2: https://drafts.csswg.org/css-easing-2/#funcdef-linear
+      // In such a case, we still want to extract the function parameters
+      if (dfn.textContent.trim().match(/^[a-zA-Z_][a-zA-Z0-9_\-]+\([^\)]+\)$/)) {
+        const fn = dfn.textContent.trim().match(/^([a-zA-Z_][a-zA-Z0-9_\-]+)\([^\)]+\)$/)[1];
```

Comment:
```suggestion
        const fn = matchFunctionDefinition[1];
```
regexp are painful enough to read, so I suggest DRYing them

Reply:
Done. FYI, the first and second regexps were different because the second one used a capturing group while the first one did not. The same logic also appeared elsewhere in the code.